appstore_tools/actions/publish.py

```diff
@@ -123,6 +123,16 @@ def publish_screenshot(
     _, file_name = os.path.split(screenshot_path)
     file_stat = os.stat(screenshot_path)
 
+    _, screenshot_ext = os.path.splitext(file_name)
```

Comment:
Let's not hard-code the extensions into `publish_screenshot`.  Instead, let's leave it as an option (with our list as the default set from the command-line).

My proposal for implmentation:
- Leave `publish_screenshot` unchanged.  If the callers tell this function to publish a path, it does it, no complaints.
- Add a parameter to `publish_screenshots` for the extension filter list.  Callers can use this if they want to filter by extension.
- Add an option to the command-line args, with the list used here as the default value.  This let's users of the command-line program override if needed via args or the config file that works with all command-line args.

Reply:
can now use `--asset-ignore` if you have non video files you need to ignore https://github.com/bennord/appstore-tools/pull/13